index.bs

```diff
@@ -352,6 +352,38 @@ An attribution source is a [=struct=] with the following items:
 :: An [=attribution filter map=].
 : <dfn>debug key</dfn>
 :: Null or a non-negative 64-bit integer.
+: aggregatable source
+:: An [=ordered map=] whose [=map/key|keys=] are [=strings=] and [=map/value|values=] are
+    non-negative 128-bit integers.
+
+</dl>
+
+<h3 dfn-type=dfn>Attribution aggregatable trigger data</h3>
+
+An attribution aggregatable trigger data is a [=struct=] with the following items:
+
+<dl dfn-for="attribution aggregatable trigger data">
+: key piece
+:: A non-negative 128-bit integer.
+: source keys
+:: An [=ordered set=] of [=strings=].
+: filters
+:: An [=attribution filter map=].
+: not_filters
+:: An [=attribution filter map=].
+
+</dl>
+
+<h3 dfn-type=dfn>Attribution aggregatable trigger</h3>
+
+An attribution aggregatable trigger is a [=struct=] with the following items:
+
+<dl dfn-for="attribution aggregatable trigger">
+: trigger data
```

Comment:
```suggestion
: <dfn>trigger data</dfn>
```

Reply:
Will add this when adding histogram creation algorithm.